### DISCUSSION TITLE
[STUDY GROUP]: GitHub Advanced Security Exam

### DISCUSSION THREAD

**Discussion by v-fidelusaleksander** [v-fidelusaleksander/ghcertified#164] [Study Together!]
# GitHub Advanced Security :book: STUDY GROUP

Hello everyone,

Many of us are preparing for the GitHub Certification Exams and I thought it would be beneficial to start a discussion where we can share:

-  experiances
-  study tips
-  ask questions 
- anything else regarding any of the topics covered in the exam 

Looking forward to hearing from you all! :rocket:

**Comment by tfsjohan** [comment]
How similar are the community added questions on GH Certified to the actual certification?

**Reply by v-fidelusaleksander** [reply]
It's hard to say, as I have no comparison metrics other than my personal exam experiance.

However..:

1. Most of the questions on this project were written by me and then @garysassano, and both of us wrote them **BEFORE** taking the exams, in the form of preparing for the exam. So don't expect questions on http://ghcertified.com/  to be 1:1 representation of what you will be asked on the exam, because we took the exam AFTER writing the questions
2. Questions on http://ghcertified.com/ cover the same topics as the exam - the ones from the official study guides. If you understand the topics, then you should be able to pass both.
There is no point to memorize questions on http://ghcertified.com/ , in fact personally I would leave taking the practice exams for the last bit of preparation. Take courses/read documentation and PRACTICE first, then practice test taking

This comes from someone that wrote most of the current question base here - and I would be really interested to know the opinion of other people on the similarities
